src/build/build_helpers.jl

```diff
@@ -239,24 +245,12 @@ function get_jump_variable_for_aux(jm, aux::AuxRef)
     end
 end
 
-function get_jump_variable_for_aux(jm, a::ScalarAux)
-    return jm[get_name(a)]
-end
-
-function get_jump_variable_for_aux(jm, a::IndexedAux)
-    return jm[get_name(a)][a.subindex]
-end
-
-function get_jump_variable_for_intermediate_supply(jm, output)
-    return jm[get_comp_supply_name(output)]
-end
-
-function get_jump_variable_for_intermediate_demand(jm, input)
-    return jm[get_comp_demand_name(input)]
-end
-
-function get_jump_variable_for_final_demand(jm, demand)
-    return jm[get_final_demand_name(demand)]
+function get_jump_variable_for_implicitvar(jm, im::ImplicitvarRef)
+    # if im.subindex===nothing    
+        return jm[get_name(im)]
+    # else
+        # return jm[get_name(im)][im.subindex]
+    # end
```

Comment:
I left in lines for indexed implicitvar, as a note in case they're necessary, but I believe they aren't. We create them, so any indexing is from the variables that enter in to their names, and already unrolled.